Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    pseudo_date (0.1.6)
+    pseudo_date (0.2.0)
```

Comment:
Don't bump gem versions bc it messes with releases.

Reply:
I didn't touch that file by hand, I think it updated itself. Maybe during `bundle install`?

Reply:
In any case, don't commit changes to it.